src/components/widgets/spoolman/SpoolSelectionDialog.vue

```diff
@@ -1,205 +1,201 @@
-          class="pt-4"
-        >
+  <app-dialog
+    v-model="open"
+    scrollable
+    :max-width="isMobileViewport ? '90vw' : '75vw'"
+    title-shadow
+  >
+    <template #title>
+      <span class="focus--text">{{ $t('app.spoolman.title.spool_selection') }}</span>
+
+      <v-spacer />
+
+      <v-menu
+        v-if="cameras.length > 1"
+        left
+        offset-y
+        transition="slide-y-transition"
+      >
+        <template #activator="{ on, attrs, value }">
           <app-btn
-            v-if="spoolmanURL"
-            :href="spoolmanURL"
-            target="_blank"
+            v-bind="attrs"
+            small
+            class="ms-1 my-1"
+            v-on="on"
           >
+            <v-icon
+              class="mr-1"
+              small
+            >
+              $camera
+            </v-icon>
+            <template v-if="!isMobileViewport">
+              {{ $t('app.spoolman.btn.scan_code') }}
+            </template>
             <v-icon
               small
-              class="mr-2"
+              class="ml-1"
+              :class="{ 'rotate-180': value }"
             >
-              $edit
+              $chevronDown
             </v-icon>
-            {{ isMobileViewport ? '' : $tc('app.spoolman.btn.manage_spools') }}
           </app-btn>
-
-          <v-spacer />
-
-          <app-btn
-            text
-            color="warning"
-            @click="open = false"
+        </template>
+        <v-list dense>
+          <v-list-item
+            v-for="camera in cameras"
+            :key="camera.id"
+            @click="scanSource = camera.id"
           >
-            {{ $t('app.general.btn.cancel') }}
-          </app-btn>
-          <app-btn
-            color="primary"
-            @click="handleSelectSpool"
+            <v-list-item-icon>
+              <v-icon>
+                $camera
+              </v-icon>
+            </v-list-item-icon>
+            <v-list-item-content>
+              <v-list-item-title>
+                {{ camera.name }}
+              </v-list-item-title>
+            </v-list-item-content>
+          </v-list-item>
+        </v-list>
+      </v-menu>
+
+      <app-btn
+        v-else-if="cameras.length"
+        small
+        class="ms-1 my-1"
+        @click="cameraScanSource = cameras[0].id"
+      >
+        <v-icon
+          class="mr-1"
+          small
+        >
+          $camera
+        </v-icon>
+        <template v-if="!isMobileViewport">
+          {{ $t('app.spoolman.btn.scan_code') }}
+        </template>
+      </app-btn>
+    </template>
+
+    <v-toolbar dense>
+      <v-spacer />
+
+      <app-column-picker
+        key-name="spoolman"
+        :headers="headers"
+      />
+
+      <v-text-field
+        v-model="search"
+        outlined
+        dense
+        single-line
+        hide-details
+        append-icon="$magnify"
+        style="max-width: 360px"
+        class="mx-2"
+      />
+    </v-toolbar>
+
+    <v-card-text>
```

Comment:
```suggestion
    <v-card-text class="pt-0">
```

The file system tables don't have any padding here, so we might want to remove it here too?
![image](https://github.com/matmen/fluidd/assets/25269274/3e584759-a6a0-40f1-9db4-e5e792091b32)


Before
![image](https://github.com/matmen/fluidd/assets/25269274/0ceee994-72a6-4a39-ab1c-94decc3a1d40)
After
![image](https://github.com/matmen/fluidd/assets/25269274/2a4b4538-c39c-4294-bcf9-04df9c81c26c)

Reply:
Agreed!